.gitignore

```diff
@@ -18,6 +18,7 @@ package-lock.json
 .env.development.local
 .env.test.local
 .env.production.local
+.vscode/*
```

Comment:
```suggestion
```

Files specific to the editor/IDE you are using should go into your local .gitignore under ~/.gitignore